Tanasee/Tanasee1.xml

```diff
@@ -33,16 +36,96 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <idno>Kǝbrān 1</idno>
                         </altIdentifier>
                     </msIdentifier>
-                    <msContents>
-                        <summary>Gospels</summary>
-                    </msContents>
                     <physDesc>
                         <objectDesc form="Codex">
-                            <supportDesc>
-                                <support><p/></support>
-                            </supportDesc>
+                        <supportDesc> 
+                            <support>
+                                <material key="parchment"/>
+                            </support>
+                            <extent>
+                                <measure unit="leaf" quantity="239">239</measure>
+                                <note>Number of leaves contrary to <bibl><ptr target="bm:Hammerschmidt1973Tanasee"/><citedRange
+                                    unit="page">84</citedRange></bibl>, who counted 240 leaves.</note>
```

Comment:
because he skipped 204?

Reply:
The last correct mark is on f. 195r. The next mark with the indication 205r is infact on 204r. So he omitted 204 or any other folio in between. I do not know, how this can be expressed shortly and precisely. Maybe it is sufficient to explain, he omitted 204 from counting.